lib/_emerge/JobStatusDisplay.py

```diff
@@ -294,3 +297,7 @@ def _display_status(self):
             if hostname is not None:
                 title_str = "%s: %s" % (hostname, title_str)
             xtermTitle(title_str)
+
+    def set_current_pkg(self, pkg):
+        pkg = str(pkg).split(',')[0][1:]
```

Comment:
Could you explain that requirement a bit more? At the moment it will set the package name from the current scheduler task. I thought we're only ever running a task for one package at a time but perhaps I've missed something.

Have updated the PR elsewhere to address the line character limit and concerns about the pkg name being spammed.

Reply:
There's `--jobs`. My rough idea was to take `i = int(time.time() % no_pkgs)` and display `pkgs[i]`, or something like that. Then the displayed package would naturally switch with clock ticks.

Reply:
Unfortunately the scheduled packages don't seem to be stored in a data structure where they could be easily identified by something like an index.

I've tried to implement something similar to this where latency associated with a job running causes the truncated name of the package associated with that job to be displayed.

Currently set to 1 second (since that made it easier for me to debug) but could be raised to 2 seconds as per the original spec.

Reply:
Here is an image example of the current output:

![2021-12-22-093002_576x377_scrot](https://user-images.githubusercontent.com/21952585/147070199-10a7ae11-4638-40d5-b02a-31560d1868b5.png)